src/riscv-integration.adoc

```diff
@@ -316,10 +316,8 @@ two tag bits in memory.
 === Zicsr, Control and Status Register (CSR) Instructions
 
 {cheri_base_ext_name} requires that RISC-V CSRs intended to hold addresses,
-like <<mtvec>>, are now able to hold capabilities. Therefore, such registers are
-removed in {cheri_base_ext_name} and analogous CLEN-bit
-versions of those CSRs are added to the ISA as described in
-xref:csr-numbers-section[xrefstyle=short].
+like <<mtvec>>, are now able to hold capabilities. Therefore, such registers
+renamed and extended to CLEN-bit in {cheri_base_ext_name}.
```

Comment:
This should say: "such registers _are_ renamed and extended..."?

Reply:
will commit fix